### PR TITLE
Fixing naming issues

### PR DESCRIPTION
**Pull request by ljakimczuk**
Towards: https://github.com/giantswarm/giantswarm/issues/23247

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,7 +58,13 @@ Support for these methods are not yet officially supported, but may still work:
 1. [Using our web interface](https://docs.giantswarm.io/ui-api/web/app-platform/#installing-an-app)
 2. [Using our API](https://docs.giantswarm.io/api/#operation/createClusterAppV5)
 
-**Impoortant Note:** If you are not using `kubectl gs` plugin, plese remember to ensure the correct label: `app-operator.giantswarm.io/version: 0.0.0` is set on the App CR. Missing this configuration will result with stuck deployment of an app.
+### **Important**
+
+If you are not using `kubectl gs` plugin, plese remember to ensure the correct label: `app-operator.giantswarm.io/version: 0.0.0` is set on the App CR. Missing this configuration will result with stuck deployment of an app.
+
+When naming the App CR, please make sure the name is original within the Management Cluster, using the sole `security-pack`
```
Comment (piontec) [inline README.md:65]:
```suggestion
When naming the App CR, please make sure the name is unique within the Management Cluster, using just `security-pack`
```

**Review by piontec — APPROVED** [review]
LGTM, with 1 improvement

**Comment by stone-z** [comment]
I'm fine with this as a temporary mitigation, is there an issue already for a more resilient long-term solution?

**Comment by ljakimczuk** [comment]
@stone-z hmm I don't believe there is any better solution to this, it is like installing the same app twice or more in the workload cluster - user must give them unique names. The same applies when installing something in the management cluster, if one installs more instances of one and the same app, he should name them uniquely.

**Comment by ljakimczuk** [comment]
Maybe we could prefix Chart CRs names with the namespaces the corresponding App CRs are coming from in the app operator, but it's hard for me to predict how good it would work, potential issues it could introduce, etc.

Also when we migrate to CAPx and org-namespaced apps for Workload Clusters, these names will have to be unique then anyway, so I see it as good step in adapting the habbit of correctly naming apps.

**Comment by ljakimczuk** [comment]
@stone-z I will drop a note for ourselves so that we discuss this in the next week's planning, will get back to you

**Comment by stone-z** [comment]
@ljakimczuk yeah happy to chat about it as well, I'm just thinking if it wasn't clear to me that the namespace separation isn't enough to distinguish these apps from each other, it probably won't be clear to customers either. So especially as more customers adopt gitops setups we will likely need to provide explicit guidance that they can't just have a "standard/base" app and will need to template a cluster ID in there as well

**Comment by ljakimczuk** [comment]
@stone-z right, we definitely need a strong guidance to make it more clear. I mean the concept can be misleading because it works on two levels, it first involves in-cluster app, hence namespace separation has no effect, and then it involves Workload Cluster apps with the namespace separations, etc. I've already left a note for Honeybadgers to discuss, if you want to discuss it as well let's maybe schedule something for tomorrow. I've conducted detailed investigation today, so I should be able to discuss the details.

**Pull request merged**